code/obj/railing.dm

```diff
@@ -144,6 +144,14 @@
 				user.show_text("[src] is already reinforced!", "red")
 
 	attack_hand(mob/user)
+		src.try_vault(user)
+
+	Bumped(var/mob/AM as mob)
+		. = ..()
+		if(AM.client?.check_key(KEY_RUN))
+			src.try_vault(AM)
+
+	proc/try_vault(mob/user)
 		if (railing_is_broken(src))
 			user.show_text("[src] is broken! All you can really do is break it down...", "red")
```

Comment:
this could get really spammy quick i think if it's broken, test to be sure it's not

Reply:
Right, good idea! It probably will be.

Reply:
Apparently broken railings get density 0, so you can't bump into them. (: